lib/pages/forward/selectable_chat_list_item.dart

```diff
@@ -1,97 +0,0 @@
-import 'package:fluffychat/pages/chat_list/chat_list_item_style.dart';
-import 'package:fluffychat/pages/forward/forward_item_style.dart';
-import 'package:fluffychat/pages/forward/selectable_chat_list_item_style.dart';
-import 'package:fluffychat/utils/matrix_sdk_extensions/matrix_locals.dart';
-import 'package:fluffychat/widgets/avatar/avatar.dart';
-import 'package:flutter/material.dart';
-import 'package:flutter_gen/gen_l10n/l10n.dart';
-import 'package:linagora_design_flutter/colors/linagora_sys_colors.dart';
-import 'package:matrix/matrix.dart';
-
-class SelectableChatListItem extends StatelessWidget {
```

Comment:
how to remove it? if you want, you can use `decorator` for `selectable` also

Reply:
This class is not used anywhere, I will delete it